apps/src/code-studio/pd/workshop_dashboard/reports/workshop_summary_report.jsx

```diff
@@ -119,6 +119,12 @@ export default class WorkshopSummaryReport extends React.Component {
     }, {
       property: 'workshop_name',
       header: {label: 'Workshop Name'}
+    }, {
+      property: 'on_map',
+      header: {label: 'Shown on Map'},
```

Comment:
Suggest "Open to the public" as the label here

Reply:
And the serializer TBH

Reply:
Note: this has already been the label for the [other](https://github.com/code-dot-org/code-dot-org/blob/bc6a5fbcd37236b357b776b8dea744e349fc27e9/apps/src/code-studio/pd/workshop_dashboard/reports/teacher_attendance_report.jsx#L150) report, it was just missing in this one. I think we should say something about the map, since this reflects the answer to "Should this appear on the K-5 workshop map?". How about `On Map` or `Shown on Map`?

Reply:
Shown on Map is fine now that I think about it